### DISCUSSION TITLE
Documentations for Using Apis

### DISCUSSION THREAD

**Discussion by dipcse07** [doronz88/pymobiledevice3#457] [Q&A]
In the documentation i  can not see anything on how to import and use this library directly to use in python code to get the device info data or battery info data or any other info..

Can anyone please give that documentation for those apis so that as a noob in python i can also use this library in my project  code 
 directly to get those information @doronz88 ?

**Comment by doronz88** [comment]
In general, you can always look at the implementation of the CLI commands to view the usage of each service.
For example: https://github.com/doronz88/pymobiledevice3/blob/master/pymobiledevice3/cli/apps.py#L49

**Comment by dipcse07** [comment]
Thanks for your reply I will check now..
